cpp/oneapi/dal/data/undirected_adjacency_array_graph.cpp

```diff
@@ -135,35 +102,37 @@ void convert_to_csr_impl(const edge_list<vertex_type<G>> &edges, G &g) {
     }
 
     layout->_vertex_count = max_id + 1;
-    layout->_degrees      = array<vertex_t>::zeros(layout->_vertex_count);
+    int_t *degrees        = (int_t *)malloc(layout->_vertex_count * sizeof(int_t));
```

Comment:
we still do not check the result of memory allocation. One of basic SDL reqs is not met

Reply:
This part will be changed by Olga with optimized convertion. This one is a draft version.